package/gluon-mesh-babel/files/lib/gluon/upgrade/330-gluon-mesh-babel-interfaces

```diff
@@ -13,6 +14,15 @@ if site.mesh_on_wan then
   uci:add_to_set('firewall', 'mesh_babel', 'network', 'wan')
 end
 
+uci:delete_all('babeld', 'interface')
+
+pcall(function() 
+  for interface in io.lines(interfaces) do 
+    int = string.gsub(interface,'[.-]','_')
```

Comment:
You can change this to `interface:gsub('[.-]', '_')`. Or even better, to `interface:gsub('[^%w_]', '_')`, which will catch all invalid chars.

Also, let's give the variable a better name than *int*, something like *section*. It should also be declared *local*.